src/resolveTaskFn.js

```diff
@@ -81,6 +81,10 @@ module.exports = function resolveTaskFn(options) {
     debug('%s  OS: %s; File path chunking unnecessary', symbols.success, process.platform)
     return () =>
       execLinter(bin, args, execaOptions, pathsToLint).then(result => {
+        if (verboseMode) {
+          console.log(result.stdout)
```

Comment:
All linters should print to stdout without this change. What exactly are you trying to solve?

Reply:
Debug mode doesn't show stdout for commands, this flag allows us to view stdout. Can be usefull.